lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0",
```

Comment:
is this solely to indicate the config version, or is it used for something else?